plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -57,6 +59,17 @@ public void execute(IProgressMonitor monitor) throws InvocationTargetException,
     AppEngineStandardFacet.installAppEngineFacet(
         facetedProject, true /* installDependentFacets */, monitor);
     AppEngineStandardFacet.installAllAppEngineRuntimes(facetedProject, true /* force */, monitor);
+    
+    setProjectIdPreference(newProject);
+  }
+
+  void setProjectIdPreference(IProject project) {
+    String projectId = config.getAppEngineProjectId();
+    if (projectId != null && !projectId.isEmpty()) {
+      IEclipsePreferences preferences = new ProjectScope(project)
+          .getNode("com.google.cloud.tools.eclipse.appengine.deploy");
```

Comment:
Is this to avoid depending on the `c.g.c.t.e.appengine.deploy` bundle? Then I'd rather move the preference node IDs into the `c.g.c.t.e.util` bundle and refer to it from everywhere else.